vue/sbc-common-components/src/services/keycloak.services.ts

```diff
@@ -116,6 +116,7 @@ class KeyCloakService {
               resolve()
             }
             redirectUrl = redirectUrl || `${window.location.origin}${process.env.VUE_APP_PATH}`
+            redirectUrl = `https://logon7.gov.bc.ca/clp-cgi/logoff.cgi?returl=${redirectUrl}&retnow=1`
```

Comment:
I was hoping we could make this configurable somewhere that way we could change it disable it?  Any ideas?